include/dxc/HLSL/DxilMetadataHelper.h

```diff
@@ -291,6 +291,9 @@ class DxilMDHelper {
   void LoadDxilCBuffer(const llvm::MDOperand &MDO, DxilCBuffer &CB);
   llvm::MDTuple *EmitDxilSampler(const DxilSampler &S);
   void LoadDxilSampler(const llvm::MDOperand &MDO, DxilSampler &S);
+  DxilResourceBase LoadDxilResourceBaseFromMDNode(llvm::MDNode *MD);
```

Comment:
Can we make this symmetric with the other functions and take a `DxilResourceBase&` operand.